docs/dapps/build-dapp-from-scratch.md

```diff
@@ -0,0 +1,548 @@
+---
+sidebar_position: 50
```

Comment:
Should we put this under `built-with-nextjs`?

Reply:
yep, I'll remove the `#Dapps Getting started`.

For the position in the side bar I don't have a strong opinion. I'll make the change in 1 commit so it's easy to revert if we think the position is not good